consensus/common/src/main/java/tech/pegasys/pantheon/consensus/common/VoteTallyUpdater.java

```diff
@@ -30,15 +26,17 @@
  * Provides the logic to extract vote tally state from the blockchain and update it as blocks are
  * added.
  */
-public class IbftVoteTallyUpdater implements VoteTallyUpdater {
+public class VoteTallyUpdater {
 
   private static final Logger LOG = LogManager.getLogger();
-  private static final Address NO_VOTE_SUBJECT = Address.wrap(BytesValue.wrap(new byte[20]));
 
   private final EpochManager epochManager;
+  private final VoteBlockInterface serialiserDeserialiser;
```

Comment:
We should be using US spelling, so `serializerDeserializer` but I'd be tempted to call this `blockInterface` or something slightly more specific.

Reply:
done before you asked.